doc/interop-test-descriptions.md

```diff
@@ -293,16 +297,14 @@ Procedure:
 
 Asserts:
 * call was successful
-* received SimpleResponse.username equals FLAGS_default_service_account
-* received SimpleResponse.oauth_scope is in FLAGS_oauth_scope
+* received SimpleResponse.username equals --default_service_account
+* received SimpleResponse.oauth_scope is in --oauth_scope
```

Comment:
Currently the server returns xapi.zoo instead of the full link. I think we can change the server to return the full link.